LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-not-001.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>:host combined with :not</title>
+<link rel="help" href="https://drafts.csswg.org/selectors/#featureless">
+<link rel="match" href="/css/reference/ref-filled-green-100px-square-only.html">
+<p>Test passes if there is a filled green square.</p>
+<div id=host>
+</div>
+<script>
+  host.attachShadow({mode: "open"}).innerHTML = `
+    <style>
+      :host {
+        width: 100px;
+        height: 100px;
+        background-color: green;
+      }
+      :host:not(:defined) {
+        background-color: red;
+      }
+      :host:defined {
+        background-color: purple;
+      }
```

Comment:
Is there benefit here for the valid selectors of `:host(:not(:defined))` and `:host(:defined)` to be placed under test with this change? Maybe they're already covered elsewhere?

Reply:
No real reason to use `:defined` indeed, will change it